integration-tests/general/default-enclave/build.gradle

```diff
@@ -15,4 +15,5 @@ conclave {
     productID = 1
     revocationLevel = 0
     runtime = runtimeType
+    useInternalDockerRepo = true
```

Comment:
I just learnt that the correct term is "registry", so can you rename this to `useInternalDockerRegistry`.